extensions/import/src/wizard/pages/modifyColumnsPage.ts

```diff
@@ -160,13 +177,13 @@ export class ModifyColumnsPage extends ImportPage {
 	}
 
 	private async populateTable() {
-		let data: ColumnMetadataArray[] = [];
+		let data: azdata.DeclarativeTableCellValue[][] = [];
 
 		this.model.proseColumns.forEach((column) => {
 			data.push(ModifyColumnsPage.convertMetadata(column));
 		});
 
-		this.table.updateProperties({
+		this.table.updateProperties(<azdata.DeclarativeTableProperties>{
```

Comment:
One of these days it'd be nice to make component generic as well for stuff like the properties so users don't have to do this casting themselves (and make it safer to boot)